hudi-client/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -103,6 +104,8 @@
   private static final String DEFAULT_MAX_COMMITS_TO_KEEP = "30";
   private static final String DEFAULT_MIN_COMMITS_TO_KEEP = "20";
   private static final String DEFAULT_COMMITS_ARCHIVAL_BATCH_SIZE = String.valueOf(10);
+  // Do not read more than 6MB at a time (6MB observed p95 in prod from 2 datasets)
```

Comment:
Please remove comments about "prod" and leave a generic comment.

Reply:
will do